epochStart/metachain/sovereignTrigger.go

```diff
@@ -36,6 +41,77 @@ func (st *sovereignTrigger) SetProcessed(header data.HeaderHandler, body data.Bo
 	st.baseSetProcessed(sovChainHeader, body)
 }
 
+// RevertStateToBlock will revert the state of the trigger to the current block
+func (st *sovereignTrigger) RevertStateToBlock(header data.HeaderHandler) error {
```

Comment:
I suggest you can add `sovereignTrigger.RevertStateToBlock` in all 3 log messages in this func

Reply:
Added it